packages/mobile/src/screens/edit-track-screen/EditTrackScreen.tsx

```diff
@@ -83,7 +83,9 @@ const useEditTrackSchema = () => {
               .string()
               .regex(/^T-?\d{3}.?\d{3}.?\d{3}.?-?\d$/i, errorMessages.iswc)
               .nullable()
-          )
+          ),
+          cover_original_song_title: z.optional(z.string().nullable()),
+          cover_original_artist: z.optional(z.string().nullable())
```

Comment:
I'm wondering if these need to be a little more nuanced. Is it a valid state to have only one of them specified? Are empty strings allowed? Are we using a separate boolean to track if "is a cover" has been enabled? If so, probably want to use `refine` to check that these both have values in that case.

Reply:
yea only one of them can be specified as a valid case. if `isCover` is true and the value is nullable, we transform it to an empty string pre submission in the `onSubmit`